pkg/metrics/metrics.jsx

```diff
@@ -815,6 +845,18 @@
                             })
                         }
                         </div>
+
+                        { this.state.topServicesDiskIO.length > 0 &&
+                            <Table
+                                variant={TableVariant.compact}
+                                gridBreakPoint={TableGridBreakpoint.none}
+                                borders={false}
+                                aria-label={ _("Top 5 services disk IO") }
+                                cells={ [{ title: _("Service"), transforms: [cellWidth(60)] }, _("Read"), _("Write")] }
+                                rows={this.state.topServicesDiskIO}>
+                                <TableHeader />
+                                <TableBody />
```

Comment:
These 9 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18070-20221222-100304-73185077-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)